Makefile

```diff
@@ -1,4 +1,5 @@
-#
+PACKAGES:=$(shell go list ./... | grep -v /vendor/)
```

Comment:
Repeating this operation on every execution of make slows down the build. It is only by some targets, we shouldn't incur this penalty for every target.

 This currently takes ~1.6s on my system, but this will only get slower.